<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainRDV</class>
 <widget class="QMainWindow" name="MainRDV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire de Rendez-vous</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="appointmentsView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>851</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>459</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="jourEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>499</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="moisEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="anneeEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>579</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>320</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;bonjout&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Ajouter Un Rendez-vous</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>430</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Mettre A Jour le RDV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>530</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
   <widget class="QComboBox" name="adresseComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>661</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Selectionner...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tunis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Djerba</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mannouba</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>JOUR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>MOIS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Ann√©e</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Heure</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="heureEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>620</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="afficherButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>630</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdfButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>720</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Export As PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statsButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>810</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Statistique</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>220</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
color:white;
background-color:red;
font: 700 8pt &quot;Arial&quot;;
border-radius:90px;
}
QPushButton:Hover
{
	background-color:black;
}</string>
    </property>
    <property name="text">
     <string> TRI Par ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>230</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>200</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Recherche Par Id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Vehicules">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
    background-color: white;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 0, 0);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Vehicules</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Services">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
    background-color: white;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 0, 0);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Services</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RDV_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
    background-color: white;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 0, 0);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>EQUIPEMENT</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>10</y>
      <width>461</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>-5</number>
    </property>
    <property name="text">
     <string>Rendez-Vous</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gestion Des Rendez-Vous</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>820</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="statsButton_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>900</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Afficher Sur l'ecran</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statsButton_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>740</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: Red;
    color: rgb(255, 255, 255);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 255, 255);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Importer Les donn√©e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RDV_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 16pt &quot;MS Shell Dlg 2&quot;;
    background-color: white;
    color: rgb(0, 0, 0);
    border: 2px solid rgb(255, 0, 0); 
    border-radius: 20px; 
    padding: 10px; 
    transition: background-color 0.3s, color 0.3s; 
}
QPushButton:hover {
    background-color: rgb(255, 0, 0);
    color: black;
    border: 4px solid rgb(255, 255, 255); 
    outline: none; 
}</string>
    </property>
    <property name="text">
     <string>Employ√©e</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRendez_Vous">
    <property name="title">
     <string>Rendez-Vous</string>
    </property>
   </widget>
   <addaction name="menuRendez_Vous"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
